src/main/java/com/openlattice/chronicle/ChronicleStudyApi.java

```diff
@@ -180,12 +238,14 @@ public Integer updateAppsUsageAssociationData(
     /**
      * Get all apps usage data associated with a participant filtered by current date
      *
-     * @param studyId       - the studyId
-     * @param participantId - the participant
+     * @param organizationId - Id of the organization to which study belongs
+     * @param studyId        - the studyId
+     * @param participantId  - the participant
      * @return a list of neighbor entities and associations
      */
-    @GET( BASE + PARTICIPANT_PATH + DATA_PATH + STUDY_ID_PATH + PARTICIPANT_ID_PATH + APPS )
+    @GET( BASE + ORGANIZATION_ID_PATH + STUDY_ID_PATH + PARTICIPANT_ID_PATH + APPS )
```

Comment:
This looks like a breaking change.

Reply:
https://github.com/openlattice/chronicle-api/pull/35#discussion_r485306813